assets/js/core/bootstrap-4.6.0.bundle.min.js

```diff
@@ -0,0 +1,7 @@
+//# sourceMappingURL=bootstrap.bundle.min.js.map
```

Comment:
Also meant to mention this one
```suggestion
//# sourceMappingURL=bootstrap-4.6.0.bundle.min.js.map
```